MSAL/test/unit/MSALPublicClientApplicationTests.m

```diff
@@ -1363,6 +1539,81 @@ - (void)testRemoveAccount_whenAccountExists_shouldRemoveAccount
     XCTAssertEqual([application allAccounts:nil].count, 0);
 }
 
+- (void)testRemoveAccount_whenAccountExists_andIsFociClient_shouldRemoveAccount_andMarkClientNonFoci
+{
+    // 1. Save response for a different clientId
+    NSString *authorityUrl = @"https://login.microsoftonline.com/0287f963-2d72-4363-9e3a-5705c5b0f031";
+    MSIDAADV2TokenResponse *msidResponse = [self msalDefaultTokenResponseWithAuthority:authorityUrl familyId:@"1"];
+    MSIDConfiguration *configuration = [self msalDefaultConfigurationWithAuthority:authorityUrl];
+
+    NSError *error = nil;
+    BOOL result = [self.tokenCacheAccessor saveTokensWithConfiguration:configuration
+                                                              response:msidResponse
+                                                               factory:[MSIDAADV2Oauth2Factory new]
+                                                               context:nil
+                                                                 error:&error];
+    XCTAssertTrue(result);
+    XCTAssertNil(error);
+
```

Comment:
nit: count perhaps as that is more significant in testing the rest of the flow?

Reply:
Addressed